en/deploy/README.md

```diff
@@ -4,243 +4,301 @@
 
-Create a file named `.gitignore` in your `djangogirls` directory with the following content:
+## Pulling our code down on PythonAnywhere
 
-    myvenv
-    __pycache__
-    staticfiles
-    local_settings.py
-    db.sqlite3
-    *.py[co]
+When you've signed up for PythonAnywhere, you'll be taken to your dashboard or "Consoles" page.  Choose the option to start a "Bash" console -- that's the PythonAnywhere version of a console, just like the one on your PC
 
-and save it. The dot on the beginning of the file name is important! As you can see, we're now telling Heroku to ignore `local_settings.py` and don't download it, so it's only available on your computer (locally).
+> __Note__: PythonAnywhere is based on Linux, so if you're on Windows, the console will look a little different from the one on your computer.
 
-Next, we’ll create a new git repository and save our changes.
+Let's pull down our code from github onto PythonAnywhere by creating a "clone" of the repo.  Type this into the console on PythonAnywhere:
 
-> __Note__: Check out your current working directory with a `pwd` command before initializing the repository. You should be in the `djangogirls` folder.
+    $ git clone https://github.com/<your-github-username>/my-first-blog.git
 
-Go to your console and run these commands:
+This will pull down a copy of your code onto PythonAnywhere.  Check it out by typing:
 
-    $ git init
-    Initialized empty Git repository in ~/djangogirls/.git/
-    $ git config user.name "Your Name"
-    $ git config user.email you@example.com
+    $ tree my-first-blog
+    my-first-blog/
+    ├── blog
+    │   ├── __init__.py
+    │   ├── admin.py
+    │   ├── migrations
+    │   │   ├── 0001_initial.py
+    │   │   └── __init__.py
+    │   ├── models.py
+    │   ├── tests.py
+    │   └── views.py
+    ├── manage.py
+    └── mysite
+        ├── __init__.py
+        ├── settings.py
+        ├── urls.py
+        └── wsgi.py
 
-Initializing the git repository is something we only need to do once per project.
 
-It's a good idea to use a `git status` command before `git add` or whenever you find yourself unsure of what will be done, to prevent any surprises from happening (e.g. wrong files will be added or commited). The `git status` command returns information about any untracked/modifed/staged files, branch status and much more. The output should be similar to:
+### Creating a virtualenv on PythonAnywhere
 
-    $ git status
-    On branch master
+Just like you did on your own computer, you can create a virtualenv on PythonAnywhere.  In the Bash console, type:
 
-    Initial commit
+    20:20 ~ $ cd my-first-blog
 
-    Untracked files:
-      (use "git add <file>..." to include in what will be committed)
+    20:20 ~ $ virtualenv --python=/usr/bin/python3.4 myvenv
```

Comment:
`virtualenv --python=python3.4 myvenv` should work too and is less typing :)

By the way, does pythonanywhere use Debian/Ubuntu (otherwise we could use `python3.4 -m venv`)?